index.js

```diff
@@ -68,7 +68,7 @@ function PinoColada () {
     if (!obj.name) obj.name = ''
     if (!obj.ns) obj.ns = ''
 
-    output.push(formatDate())
+    output.push(formatDate(obj.time || null))
```

Comment:
You must not use `null` because `new Date(null)` is equal to `new Date(0)` which is "Thu Jan 01 1970 00:00:00". But you can use `Date.now()` to get current timestamp.

```suggestion
    output.push(formatDate(obj.time || Date.now()))
```